packages/surveys/src/lib/logic.ts

```
@@ -0,0 +1,473 @@
+  jumpTarget: string | undefined;
+  requiredQuestionIds: string[];
+  calculations: TResponseVariables;
+} => {
+  let jumpTarget: string | undefined;
+  const requiredQuestionIds: string[] = [];
+  const calculations: TResponseVariables = { ...calculationResults };
+
+  actions.forEach((action) => {
+    switch (action.objective) {
+      case "calculate":
+        const result = performCalculation(survey, action, data, calculations);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)